build/configure

```diff
@@ -130,7 +131,7 @@ fi
 
 (cd $scxpal_dir/build/ && chmod ug+x ./configure; ./configure $enable_debug $enable_bullseye $enable_ulinux_qual)
 
-omi_configure_quals="--enable-preexec ${enable_debug} ${enable_purify_agent} ${enable_purify_server} ${omi_configure_quals}"
```

Comment:
I was wondering where --enable-preexec went, it looks like it's enabled by the new --enable-microsoft flag which also encompasses other flags. Seems fine.